### ISSUE TITLE
Create RW transport taskings

### ISSUE THREAD

**Issue by 132ndNeck** [132nd-vWing/SDCM#6]
Create some RW transport taskings. 
Create a logistical base, not far from KUTAISI, and one not far away from SENAKI. The idea is to have the MI-8 fly out supplies from these logistical bases to friendly troops to support the mission.
Likely use the CTLD script in order to have an impact on the mission itself.

Question1 : Is it possible to make the taskings dynamic in a way?
Question 2: Is there cargos that can be transported in DCS , or should we also include this mod with the cargo that can be lifted by the MI-8?

**Comment by theendsofinvention** [comment]
Pointers:

1) http://flightcontrol-master.github.io/MOOSE/Documentation/Mission.html
2) http://flightcontrol-master.github.io/MOOSE/Documentation/Cargo.html

**Comment by theendsofinvention** [comment]
This also looks great: http://flightcontrol-master.github.io/MOOSE/Documentation/Assign.html

**Comment by 132nd-Entropy** [comment]
should be easily doable, I can try it. Before I start, just to make sure I understand: 
We create the logistical base (you want this always in the same place, or look like a makeshift base and be in a slightly different location every time (but always close to KUTAISI, and not far from SENAKI). 
So the helos would take off and find the base where the cargo is (smoke, radio beacon) and then drop it off at the target (where there will be some tents or units). Basically just ferry crates back and forth? 

Once they unload a crate, should that increase a counter, or spawn a text message? (or spawn units to reinforce the troops already there)

**Comment by 132ndNeck** [comment]
I would suggest making a logistical base up in the vincinty of the mini airfield at Kutaisi. There we could place different objects, and also place static units to look like a base. So this base will stay in place at the same area everytime.

My ideas was that the MI-8 pilot "check-in" using the F10 menu, asking for a tasking. When a MI-8 pilot does so, he can either be tasked to pick-up something from the ammo depot at either Senaki airfield, Kutaisi airfield or the logistical base at Kutaisi. At the same time, some friendly units gets spawned in somewhere, which will then be the "target" for the supply run. This should not be too far away from Kutaisi, as the fligth for the helo's with slingload should not take more than 15 minutes(based on feedback from the Mi-8 pilots on Discord). 

So, yes, basically ferry crates back and forth. When they unload a crate, it can either be supplies, or personell to reinforce the troops already in place. 

Once that is done, the MI-8 pilot can check-in again for new tasking.

**Comment by 132nd-Entropy** [comment]
got it, sounds great, Im on it!

**Comment by theendsofinvention** [comment]
Is this still a thing ?

**Comment by 132nd-Entropy** [comment]
yes but I was busy and had no time. Plus, the Mi-8 sqd is terribly inactive with both looney and makeshift more or less gone and me mainly flying fixed wing. I will get to it as soon as I can, it should be straight forward.

**Comment by theendsofinvention** [comment]
I agree, I haven't seen a Bear in ages.

I think this should be set to low priority for the time being and be re-evaluated in September.
